src/components/src/ProductDisplayItem/productdisplayitem.main.tsx

```diff
@@ -607,8 +615,15 @@ class ProductDisplayItemMain extends Component<ProductDisplayItemMainProps, Prod
       vrMode,
       vrFileExists,
       multiImages,
+      detailsProductData,
     } = this.state;
 
+    const productDescription = detailsProductData ? detailsProductData[0]['display-value'].split('. ', 1)[0] : '';
```

Comment:
This isn't quite displaying the description. I'm seeing in the Netlify deploy that it's pulling all attributes. We'll only want to display the price and first sentence of the description. Currently it's showing "True", which is the first attribute returned, whereas the description is the second attribute. 

![IMG_0850](https://user-images.githubusercontent.com/39800563/76629853-61ef1b00-6515-11ea-947e-db3170be2a89.PNG)